yarn.lock

```diff
@@ -2027,15 +2027,6 @@
     call-me-maybe "^1.0.1"
     glob-to-regexp "^0.3.0"
 
-"@near-wallet/feature-flags@^0.0.4":
```

Comment:
@MaximusHaximus not sure why this dropped the reference altogether. What is the correct way to bump the feature flag package? Do we need to be running a `lerna` command?